torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -300,9 +355,9 @@ class MKLDNNSubgraphSlicer {
       return supportedMKLDNNWeight(*const_tensor);
     }
     auto k = v->node()->kind();
-    bool supported_kind = k == prim::MKLDNNGroup || k == aten::to_mkldnn ||
-        k == prim::ConstantMKLDNNTensor;
-    if (supported_kind) {
+    bool supported = k == prim::MKLDNNGroup ||
```

Comment:
nitpick: if that "if" statement is the only use of `supported` maybe we should just inline it?